Docs: a brief introduction to OpenCV

gromag opened the pull request:
For those unfamiliar with OpenCV (particularly OpenCV Python), I have added a brief introduction to provide an overview of what OpenCV is about. 
The current README jumps straight into installation instructions without providing a clear description of the repository. While experienced OpenCV developers may not require an introduction, I believe it's important to be inclusive and help new developers understand what OpenCV is about.

Additionally, I suggest adding a brief introduction to the main OpenCV repository to improve accessibility for all users.

gromag committed:
Docs: a brief introduction to OpenCV 

Individual not familiar with OpenCV can get a quick first glance understanding of what OpenCV (and OpenCV Python in particular) is about.

asmorkalov commented on README.md:
```diff
@@ -4,6 +4,8 @@
 Pre-built CPU-only OpenCV packages for Python.
 
+OpenCV (Open Source Computer Vision Library) is an open-source computer vision and machine learning software library. OpenCV Python provides a set of easy-to-use functions for image and video processing, including image loading, filtering, transformation, feature extraction, object detection, and more. It provides interfaces to popular machine learning frameworks like TensorFlow and PyTorch, maki ...
+
```
OpenCV does not provide interfaces for Tensorflow or PyTorch. Just reference to OpenCV project itself  is enough.

gromag replied:
Thanks @asmorkalov, I'm new to OpenCV and I thought it did, I have amended the text as per your suggestion.

gromag committed:
Update README.md

Removed reference to Tensorflow and Pytorch as per Alexander Smorkalov's comment.

gromag committed:
Merge branch 'opencv:4.x' into 4.x

gromag committed:
Merge pull request #1 from opencv/4.x

updating from fork
